src/components/Alert.js

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
```

Comment:
I'm adding this here, but it applies to all files. I use dash-separate filenames and they start with a lowercase letter. For example, `Alert.js` should be `alert.js` and `PeopleDrawer.js` should be `people-drawer.js`.

Reply:
I used this convention as for react components it's common to use the pascal case naming convention.